packages/flutter/test/material/dropdown_test.dart

```diff
@@ -1290,9 +1290,8 @@ void main() {
     // By default the hint contributes the label.
     expect(tester.getSemantics(find.byKey(key)), matchesSemantics(
       isButton: true,
-      label: 'test',
```

Comment:
The label semantic should be tested here and below.

Reply:
Return semantics have changed after wrapping InkWell, will update after the team review this private constructor approach